app/assets/javascripts/components/overview/statistics_update_info.jsx

```diff
@@ -49,33 +31,30 @@ const StatisticsUpdateInfo = createReactClass({
   render() {
     const course = this.props.course;
 
-    if ((Features.wikiEd && !course.ended) || !course.updates.last_update) {
+    if (Features.wikiEd && !course.ended) {
       return <div />;
     }
 
-    // Render Modal
+    const [lastUpdateMessage, nextUpdateMessage, isNextUpdateAfter] = this.getUpdateTimesArray();
+
     if (this.state.showModal) {
       return (
         <StatisticsUpdateModal
           course={course}
-          getUpdateTimesInformation={this.getUpdateTimesInformation}
+          isNextUpdateAfter={isNextUpdateAfter}
+          nextUpdateMessage={nextUpdateMessage}
           toggleModal={this.toggleModal}
         />
       );
     }
 
-    const updateTimesInformation = this.getUpdateTimesInformation();
-    let updateTimesMessage = '';
-    if (updateTimesInformation !== null) {
-      const [lastSuccessfulUpdateMessage, nextUpdateMessage, isNextUpdateAfter] = updateTimesInformation;
-      updateTimesMessage = isNextUpdateAfter ? `${lastSuccessfulUpdateMessage}. ${nextUpdateMessage}. ` : `${lastSuccessfulUpdateMessage}. `;
-    }
+    const updateTimesMessage = isNextUpdateAfter ? `${lastUpdateMessage} ${nextUpdateMessage} ` : `${lastUpdateMessage} `;
 
-    // Render update time information along with 'See More' link to open modal
+    // Render update time information and if some updates were made a 'See More' link to open modal
     return (
       <div className="statistics-update-info pull-right mb2">
         <small>
-          { updateTimesMessage }<a onClick={this.toggleModal} href="#">{I18n.t('metrics.update_statistics_link')}</a>
+          {updateTimesMessage} {course.flags.first_update && <a onClick={this.toggleModal} href="#">{I18n.t('metrics.update_statistics_link')}</a>}
```

Comment:
With this guard statement, the 'see more' link does not show up for any courses already in the system before the 'first_update' flag feature gets deployed. Instead, the modal (or getUpdateTimesArray) should be updated to be able to handle any shape of `course` object.

Reply:
oh, sure. Wasn't thinking about consequences